woodwork/type_sys/utils.py

```diff
@@ -141,3 +141,32 @@ def _get_specified_ltype_params(ltype):
         # Do not reveal parameters for an uninstantiated LogicalType
         return {}
     return ltype.__dict__
+
+
+def _is_categorical(series: pd.Series, threshold: float) -> bool:
```

Comment:
should we call this `_is_categorical_series` similar to `_is_numeric_series` in the same file

might help differentiate from `ColumnSchema.is_categorical`

Reply:
Yeah, seems like a good idea.